tests/hm_typing/in_expressions.sy

```diff
@@ -0,0 +1,9 @@
+f :: fn a: *, b: * -> *
+    a in b
+end
+
+start :: fn do
+    true <=> f' 1, [1, 2, 3]
+    true <=> f' 1, {1, 2, 3}
+    false <=> f' 0, {1, 2, 3}
+end
```

Comment:
What about dicts? I feel like it would not work as expected.

Reply:
We can change the constraint, now you would check for tuples in it, unless we double the constraints.